lib/api/model/pokemon_abilities.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:pokedex_async_redux/api/model/pokemon_ability.dart';
+
+part 'pokemon_abilities.freezed.dart';
+
+part 'pokemon_abilities.g.dart';
+
+@freezed
+class PokemonAbilities with _$PokemonAbilities {
+  factory PokemonAbilities({
```

Comment:
kindly remove prefix Pokemon - Ability is enough. also don't forget to update the dart file as well 🙂